src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php

```diff
@@ -152,9 +152,6 @@ public function testLoadValuesForChoicesDoesNotLoadIfEmptyChoices()
 
     public function testLoadValuesForChoicesDoesNotLoadIfSingleIntId()
     {
-        $this->expectException(LogicException::class);
```

Comment:
we have an assertSame() below and we define the IdReder class, not sure if deleting it, is ok

Reply:
Removing the exception expectation changes the behavior of the test, so not good.

the assertSame is probably the one being dead code by having an exception triggered before reaching the assertion.

Reply:
I will double check later, which part can be removed, but one needs to be I guess

Reply:
Ok, I readded the exception. I am a bit confused about the exception message, because the idReader is explicitly defined, isn't it?

Reply:
cc @derrabus as you are doing a lot with Doctrine